app/db/index.js

```diff
@@ -4,20 +4,26 @@ const { readFileSync } = require('fs');
 const { database, server } = require('../config');
 
 const connect = () => {
-  const { name, username, password, host, dialect, port } = database;
+  const { URL, name, username, password, host, dialect, port } = database;
   const config = {
-    host,
-    dialect,
-    port,
-    logging: false,
+    connection: () => {
+      if (URL) return URL;
+      return `postgres://${username}:${password}@localhost:5432/${name}`;
+    },
+    options: {
+      host,
+      dialect,
+      port,
+      logging: false,
+    },
   };
-  if (server.environment === 'staging') {
+  if (server.environment !== 'development') {
```

Comment:
This assumes that people will set the env var to `development` -- should we assume this?

Reply:
Well, the `environment` object key is set to `development` by default in the configuration, unless another value has been set in the `.env` file.

Reply:
I could set it to `if (server.environment === 'staging')`, but if we'd put this on a production server, and set the `NODE_ENV` environmental variable to anything else than staging, the SSL configuration which should be applied by that block would not work.

Reply:
Set to `if (server.environment === 'staging')`